docs/Translation.md

```diff
@@ -343,3 +379,31 @@ translate('not_yet_translated', { _: 'Default translation' })
 ```
 
 To find more detailed examples, please refer to [http://airbnb.io/polyglot.js/](http://airbnb.io/polyglot.js/)
+
+## Using asynchronous messages loading 
+
+By default React Admin loads all configured lanuages and languagebundles on startup. If you have very large language files, you can load messagebundles asynchronous. 
+React-admin uses an `i18nProvider` to load messages. A custom dataprovider can be supplied to the `Admin` component by using the `i18nProvider` prop. 
+The signature of the `i18nProvider` is: 
+```jsx
```

Comment:
Please add a new empty line above. Otherwise, it will not render correctly.

Screenshot:

![screen](https://i.imgur.com/hjMaQJL.png)